utils/build.config

```diff
@@ -10,7 +10,7 @@ SPDK = v22.01.2
 OFI = v1.18.0
 MERCURY = v2.3.0
 PROTOBUFC = v1.3.3
-UCX=v1.13.0
+UCX=v1.14.1
```

Comment:
This isn't mentioned in the commit message, does anything else on the rpm side need a corresponding change?

Reply:
It's in the follow-on commit message.  I couldn't get 1.13.0 to compile on my fedora 38 box.  The RPM isn't actually used in any testing but there has been an RPM update in progress for a while.  My main point in doing this was that there are no compiler warnings.   Locally, I've just had WARNING_LEVEL=warn set.

Reply:
We don't use this version of UCX in testing.  It's just for the build and then MOFED is used at runtime.   So I'm not sure it matters much.  That said, @jgmoore-or does have an older update PR in progress: https://github.com/daos-stack/openucx/pull/9